Design-Docs/Image-Properties-Spec.md

```diff
@@ -211,3 +211,25 @@ A boolean property that is not present is considered to be `false`.
   pricing (but it might be provided by a contracted 3rd party, e.g. the OS vendor).
 
 
+##  Standardized Image  
+
+Images which are "default" in Glance of any SCS-Provider and are certified by SCS, should have following Features:
+
+*   Optional: Latest LTS Distribution of:
+ -    Debian
+ -    Ubuntu, 
+ -    AlmaLinux and/or Rocky Linux
+ -    OpenSLES
+
```

Comment:
what about all the other linux distro's out there? is there a good way to not have a biased list? https://distrowatch.com/dwres.php?resource=ranking&sort=average100 didn't really help me besides remembering that there is a huge Arch Linux Community out there that I tend to forget...